### PR TITLE
Add meshplex, 'Fast tools for simplex meshes.' package

### PR DESCRIPTION
**Pull request by rbeucher**
Dear @conda-forge/help-python,

Please consider adding meshplex to conda-forge

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Romain

### CODE REVIEW

**Comment by rbeucher** [comment]
The package is not compatible with python 3.8. How can I limit the version to python <3.8.
Using pinning in the meta.yml results in `Non noarch packages should have python requirement without any version constraints.`

**Comment by rbeucher** [comment]
@conda-forge/staged-recipes should be good to go

File: recipes/meshplex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "meshplex" %}
+{% set version = "0.13.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (ocefpaf) [inline recipes/meshplex/meta.yaml:6]: You can remove the quites in the jinja here and above. Also in the script line below.

File: recipes/meshplex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  url: https://github.com/nschloe/meshplex/archive/v{{ version }}.tar.gz
+  sha256: ceadd26da8c4bfdd4e543f48944b43a5c38ad4faa55563330651fb6d6f6ac765
+
+build:
+  number: 0
+  skip: true  # [py==38]
```
Comment (ocefpaf) [inline recipes/meshplex/meta.yaml:14]: Should this be `>=py38`? Or does it work with `py39`?

File: recipes/meshplex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - scipy
+    - meshio >=4,<5
+
+test:
+  imports:
+    - meshplex
+
+about:
+  home: "https://github.com/nschloe/meshplex"
+  license: GPL-3.0-only
```
Comment (ocefpaf) [inline recipes/meshplex/meta.yaml:39]: Looks like it is "or-later." See https://github.com/nschloe/meshplex/blob/master/setup.cfg#L13

File: recipes/meshplex/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/meshplex/meta.yaml:14]: Actually it does work with py38, see https://github.com/nschloe/meshplex/blob/master/setup.cfg#L35

File: recipes/meshplex/meta.yaml (hunk 3 above)
Comment (rbeucher) [reply recipes/meshplex/meta.yaml]: Yes, you are right

File: recipes/meshplex/meta.yaml (hunk 2 above)
Comment (rbeucher) [reply recipes/meshplex/meta.yaml]: OK. I have removed that

File: recipes/meshplex/meta.yaml (hunk 1 above)
Comment (rbeucher) [reply recipes/meshplex/meta.yaml]: Done

**Comment by rbeucher** [comment]
@ocefpaf I think it is good to go.

File: recipes/meshplex/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - importlib_metadata
+    - pip
+    - python
+    - numpy >=1.9
+    - scipy
+    - meshio >=4,<5
```
Comment (ocefpaf) [inline recipes/meshplex/meta.yaml:23]: You probably don't need anything but pip and python on host. Let's try that before merging this.

Comment (rbeucher) [reply recipes/meshplex/meta.yaml]: OK

**Comment by rbeucher** [comment]
@ocefpaf It looks like Travis CI failed

**Comment by ocefpaf** [comment]
> @ocefpaf It looks like Travis CI failed

We are having some issues with out infrastructure. It should be solved in a few hours. Hold on...

**Comment by rbeucher** [comment]
OK no pb!

**Pull request merged**